firmware/ci.c

```diff
@@ -231,6 +231,7 @@ static char *readstr(void)
 	char c[2];
 	static char s[128]; // Needs to fit a full mode line ~100 chars.
 	static int ptr = 0;
+	static char skip = 0;
```

Comment:
Maybe something like `skip_if_next` is a better name? 
We only skip the value if it is exactly the *next* value to be received right? IE `\ni\r` wouldn't hit the skip?

Reply:
Yes, only next.  I can change the name something like that.  Again, I was just following the naming from litex.